compiler/modules/front/expr.bal

```diff
@@ -99,6 +99,13 @@ type RegExprEffect record {|
     bir:Register result;
 |};
 
+type AtomicFunctionInfo record {|
+    t:FunctionSignature signature;
+    bir:FunctionRef|bir:Register func;
+|};
+
+type CalledFunctionInfo AtomicFunctionInfo|bir:Register;
```

Comment:
CalledFunctionInfo has bir:FunctionOperand and optional signature.

Reply:
Fixed